src/sbvr-api/permissions.ts

```diff
@@ -1666,21 +1668,18 @@ export const addPermissions = async (
 		}
 	}
 
-	// This bypasses in the root cases, needed for fetching guest permissions to work, it can almost certainly be done better though
-	let permissions = req.user?.permissions ?? [];
-	permissions = permissions.concat(req.apiKey?.permissions ?? []);
+	let permissionsLookup;
 	if (
-		permissions.length > 0 &&
-		$checkPermissions(
-			getPermissionsLookup(permissions),
-			permissionType,
-			vocabulary,
-		) === true
+		guestPermissionsInitialized === false &&
+		(req.user === root.user || req.user === rootRead.user)
```

Comment:
How about instead of this we just change the `getReqPermissions` to never fetch the guest permissions for the root users, (regardless of `guestPermissionsInitialized`)? If we are fine not having the guest permissions once for root users, then I guess we might be fine not having them at all consistently.

Reply:
I think you're right that moving it into `getReqPermissions` makes sense to move it closer to the source, however I think it makes sense to limit the special case as much as possible to avoid on-going differences between eg
```
	passthrough: {
		req: rootRead,
	},
```
and
```
	passthrough: {
		req: {
			user: {
				id: 0,
				actor: 0,
				permissions: ['resource.read'],
			},
		},
	},
```
Where the former would not include guest permissions but the latter would despite otherwise being identical